dev/com.ibm.websphere.org.eclipse.microprofile/config.1.1.bnd

```diff
@@ -26,14 +25,4 @@ Export-Package: \
 Include-Resource: \
   @${repo;org.eclipse.microprofile.config:microprofile-config-api;1.1;EXACT}
 
-
 WS-TraceGroup: APPCONFIG
-
--buildpath: \
-	com.ibm.websphere.javaee.cdi.1.2;version=latest, \
-	com.ibm.ws.org.osgi.annotation.versioning;version=latest
-
-instrument.disabled: true
-
-publish.wlp.jar.suffix: dev/api/stable
```

Comment:
Copyright dates on all these modified bnd files should be updated.

Reply:
I will update them accordingly.

Reply:
Copyright dates are updated.